draft-regext-rdap-extensions.md

```diff
@@ -140,6 +183,37 @@ extensions that may have an "author" structure. But the JSON contained
 within "lunarNIC_author" need not be prepended as the extension collision
 is avoided by "lunarNIC_author".
 
+## Suffixless Prefixes #{suffixless_prefixes}
+
+Some RDAP extensions define only one JSON value and do not prefix it with their
+RDAP extension identifier instead using the extension identifier as the JSON name
+for that JSON value. This is a "suffixless prefix".
+
+Consider the example in (#child_json_values). Using the "suffixless prefix" pattern,
+that example could be written as:
+
+    {
+      "handle" : "ABC123",
+      "remarks" :
+      [
+        {
+          "description" :
+          [
+            "She sells sea shells down by the sea shore.",
+            "Originally written by Terry Sullivan."
+          ]
+        }
+      ],
+      "lunarNIC" :
+      {
+        "firstInitial": "J",
+        "lastName": "Heinlein"
+      }
+    }
+
+The use of a "suffixless prefix" is not in accordance with the guidance given
+in [@!RFC9083] and is NOT RECOMMENDED.
```

Comment:
Given that bare extension identifiers are in use (see e.g. the redacted document), clients are going to have to take them into account during development.  If clients have to do that, then there doesn't seem to be much downside to making use of them more generally.  What about just noting that this document clarifies the existing documents in this respect, by explicitly permitting the use of such extension identifiers?

Reply:
I'm good with that too.